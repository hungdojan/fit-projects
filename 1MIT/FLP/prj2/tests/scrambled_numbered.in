341
551
225
114 166 232 645
114 122 532 645
442 555 436 433
663
663
123
