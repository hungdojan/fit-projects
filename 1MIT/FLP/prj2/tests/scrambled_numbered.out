341
551
225
114 166 232 645
114 122 532 645
442 555 436 433
663
663
123

344
555
222
111 625 332 645
111 625 332 645
445 115 336 433
664
664
122

555
555
222
111 622 333 445
111 622 333 445
445 111 622 333
664
664
664

555
555
222
111 622 333 445
111 622 333 445
111 622 333 445
444
666
666

555
555
555
111 222 333 444
111 222 333 444
111 222 333 444
666
666
666
